app/src/androidTest/java/com/github/se/wanderpals/screens/CreateSuggestionScreenTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.se.wanderpals.screens
+
+import androidx.compose.ui.test.SemanticsNodeInteractionsProvider
+import io.github.kakaocup.compose.node.element.ComposeScreen
+import io.github.kakaocup.compose.node.element.KNode
+
+class CreateSuggestionScreenTest(semanticsProvider: SemanticsNodeInteractionsProvider) :
+    ComposeScreen<CreateSuggestionScreenTest>(
+        semanticsProvider = semanticsProvider,
+        viewBuilderAction = { hasTestTag("createSuggestionScreen") }) {
+
+  val screenTitle: KNode = onNode { hasTestTag("createSuggestionTitle") }

```

Comment:
You can create a test to use the `screenTitle` in order to get a higher coverage on SonarCloud

and a detail: change the class name from `CreateSuggestionScreenTest.kt` to `CreateSuggestionScreen.kt`

Reply:
Thanks for the comment,
I changed the class name like suggested.